src/Learning/KWDataPreparation/KWDataPreparationTask.cpp

```diff
@@ -239,7 +239,8 @@ int KWDataPreparationTask::ComputeMaxLoadableAttributeNumber(const KWLearningSpe
 		if (nAttributePairNumber > 0)
 			sMessage += "and bivariate ";
 		sMessage += "data processing)";
-		AddError(sMessage + ",\n\t" + grantedResources.GetMissingResourceMessage());
+		AddError(sMessage);
+		AddError(grantedResources.GetMissingResourceMessage());
```

Comment:
If I understand correctly, the TAB is not present anymore. Is this the desired change?

Reply:
Non, ce n'est pas nécessaire
On a ici deux messages de type 
```
error : Univariate data preparation : Not enough memory for data preparation with 1,720,663 records (load data base, univariate data processing)
error : Univariate data preparation : system resources are not sufficient to run the task (need 233.7 GB of additional memory)
```
au lieu de précédemment un seul message multi-ligne de type
```
error : Univariate data preparation : Not enough memory for data preparation with 1,720,663 records (load data base, univariate data processing)
          system resources are not sufficient to run the task (need 233.7 GB of additional memory)
```
Le TAB était nécessaire pour ne pas se retrouver au début de ligne au même niveau que `error`

Reply:
OK, it's clear.